mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -12,12 +16,22 @@ use mqtt_broker::{BrokerHandle, ClientId, Error, Message, SystemEvent};
 
 const DISCONNECT_TOPIC: &str = "$edgehub/disconnect";
 
-#[derive(Debug)]
-pub struct ShutdownHandle(mqtt3::ShutdownHandle);
+pub struct ShutdownHandle(Sender<()>, mqtt3::ShutdownHandle);
```

Comment:
I tried making them named by expanding the struct as such:
```
struct foo {
  field: type,
  field2: type2,
}
```

This didn't work because I was using the parenthesis, but I will try again just to make sure. Lmk if I am missing something obvious.

Reply:
Nevermind I tried again and it works